public/locations-list-map.js

```diff
@@ -1,5 +1,49 @@
 import toDataByLocation from './toDataByLocation.js';
 
+
+
+// TODO (patricknelson): The indentation in this file is not consistent and should be updated soon. Currently the project
```

Comment:
File an issue instead of a TODO for this please.

Reply:
I can but will that hold up merge?